tests/e2e/config/config.go

```diff
@@ -67,6 +70,9 @@ func ReadE2EConfig(configPath string) (*E2EConfig, error) { //nolint:gocyclo
 
 	if chart := os.Getenv("CHART"); chart != "" {
 		config.Chart = chart
+		dir, chartFile := filepath.Split(chart)
+		chartFile = strings.Replace(chartFile, "elemental-operator", "elemental-operator-crds", 1)
+		config.CRDsChart = filepath.Join(dir, chartFile)
```

Comment:
Something like this is how I'd adapt the elemental operator chart URI to _guess_ the crds chart.

The image reference:
`rancher/elemental-operator-chart` -> `rancher/elemental-operator-crds-chart`

The release name:
`elemental-operator` -> `elemental-operator-crds`